pisa/utils/random_numbers.py

```diff
@@ -82,7 +82,7 @@ def get_random_state(random_state, jumpahead=0):
 
     """
     if random_state is None:
-        new_random_state = np.random
+        new_random_state = np.random.RandomState()
```

Comment:
FYI, it should be documented, but it was intentional to use Numpy's internal random state if `random_state` is None. We could debate whether this is a smart thing to do, but basically my reasoning was "if you do nothing, you get the behavior as if you just called `np.random.foo`." That is how many people use random numbers in Numpy, whether or not that's a good idea or something we should even allow is up for debate.

Reply:
Is `np.random.RandomState()` not "Numpy's internal random state"? I.e., I consider the above a fix that's in line with the intention you describe. Calling `get_random_state(get_random_state(None))` (which happens if you try and fluctuate a `MapSet` without passing a seed or a random state) currently fails because of a `TypeError`.

Reply:
No, that's not the case: https://numpy.org/doc/1.16/reference/generated/numpy.random.RandomState.html 
>   If seed is None, then RandomState will try to read data from /dev/urandom (or the Windows analogue) if available or seed from the clock otherwise.

There's an additional wrinkle, in that Numpy's random number generation has changed some since we first wrote PISA, so it's possible that using `np.random` directly will give different behavior from before (i.e., reference a different random number generator, nothing to do with seeding or where its state comes from), whereas the random number generator used in `np.random.RandomState()` has been guaranteed to give identical results in the future as it gave in the past.

Maybe we simply error out in this case. Why is a user doing anything without explicitly supplying a seed in PISA anyway? I can only think of sloppy, bad, and/or wrong things. Thoughts?

Reply:
Ok, good to know. I'm ok with not permitting to pass in `None` here. That would still allow the user to create some random state outside of PISA (if they must).

Reply:
I've decided against touching `get_random_state` (reverted back to its original behaviour for now) because I can't afford the time to look through the code base right now (for example, we have to adapt the docstring of `Map.fluctuate`).